ci/README.md

```diff
@@ -0,0 +1,99 @@
+[logo]: https://github.com/clems4ever/authelia/raw/master/docs/images/authelia-title.png "Authelia"
+[![alt text][logo]](https://github.com/AntilaX-3/)
```

Comment:
Why is the logo linked to this URL? Can you please either refer to authelia homepage or just remove it?

Reply:
Yep remnants from my other repo I copied the `README.md` from, fixed now though.